cadc-storage-adapter-ad/src/main/java/org/opencadc/inventory/storage/ad/AdStorageAdapter.java

```diff
@@ -144,6 +147,14 @@ public void get(StorageLocation storageLocation, OutputStream dest)
         URL sourceURL = this.toURL(storageLocation.getStorageID());
         log.debug("sourceURL: " + sourceURL.toString());
 
+        try {
+            boolean auth = CredUtil.checkCredentials();
+            log.debug("authenticated: " + auth);
+        } catch (CertificateExpiredException | CertificateNotYetValidException unexpected) {
+            log.debug("error type: " + unexpected.getClass());
```

Comment:
This has to be an AccessControlException with a clear reason... something like 

```
permission denied -- reason: delegated X509 Certificate is invalid
```

Also: it is definitely *not* unexpected and the debug message can be removed

Reply:
Changed exception type and message, and removed the debug message.